script.js

```diff
@@ -354,3 +354,19 @@ button9.className = 'button7';
 button9.innerText = p5['source button'];
 button9.appendChild(img4);
 div19.appendChild(button9);
+
+
+//Local starge
+form.addEventListener('input', () => {
+  const object = {
+    myname: myName.value,
+    myemail: email.value,
+    mymessage: message.value
+  }
+  localStorage.setItem('object', JSON.stringify(object))
+});
+
+const fetchData = JSON.parse(localStorage.getItem('object'));
+myName.value = fetchData.myname;
+email.value = fetchData.myemail;
+message.value = fetchData.mymessage;
```

Comment:
- [ ] When I tested your project locally I got this error in the console. As you can see also in the image nothing is saved in the local storage when I typed in the name input. Kindly fix the issue.
![image](https://user-images.githubusercontent.com/43698511/132093712-286b7ce6-b5b2-4bd5-955d-248bb0ffcf63.png)
Note that the expected behavior of the app is. After the user fills the form, and does not submit it, the entered values should be saved in the local storage. So that when he visits the page again the inputs will be populated with the values entered in the previous visit of the page.